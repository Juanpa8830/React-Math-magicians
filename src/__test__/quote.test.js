import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/quote';

test('fit Quote snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
