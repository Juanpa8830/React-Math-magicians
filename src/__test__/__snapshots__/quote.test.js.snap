// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match Quote snapshot 1`] = `
<div
  className="quote-section"
>
  <h2
    className="quote-title"
  >
    'Imagination is more important than knowledge.'- Albert Einstein.
  </h2>
</div>
`;
